README.md

```diff
@@ -1,9 +1,39 @@
+[comment]: # (NOTE: This file is generated and should not be modify directly. Update `templates/ROOT_README.hbs.md` instead)
 # Web foundation
 
-This repo is the living guide to how we build web UI at Shopify. It captures the [principles](Principles) we use to make decisions, our [best practices](Best%20practices) and [styleguides](Styleguides) for writing UI code, and a [log of decisions](Decision%20records) that led us to this foundation. It also provides a list of [resources](Resources) for mastering the different parts of web UI development.
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md)
+[![Build Status](https://travis-ci.org/Shopify/web-foundation.svg?branch=master)](https://travis-ci.org/Shopify/web-foundation)
+[![codecov](https://codecov.io/gh/Shopify/web-foundation/branch/master/graph/badge.svg)](https://codecov.io/gh/Shopify/web-foundation)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 
-## Contributions
+This repository contains common configuration and living guidelines that Shopify uses to build web UI. It captures the [principles](./handbook/Principles) we use to make decisions, our [best practices](./handbook/Best%20practices) and [styleguides](./handbook/Styleguides) for writing UI code, several packages that help enforce those decisions in various tools and a [log of decisions](./handbook/Decision%20records) that led us to this foundation . It also provides a list of [resources](./handbook/Resources) for mastering the different parts of web UI development.
```

Comment:
Since this is moved to `handbook/readme.md` I think we can :fire: this. Though we should leave a note linking to the handbook section.

Reply:
Suggestion, only this?

>This repository contains common configuration and living [guidelines](handbook) that Shopify uses to build web UI